README.rst

```diff
@@ -12,17 +13,45 @@ Requirements
 
 PHP 5.3.2 or higher are currently supported by these bindings.
 
-You need `mbstring <http://www.php.net/manual/en/mbstring.installation.php>`_. installed
-And Https Module Support for Curl.
+You will also need to install the non-default extensions `mbstring
+<http://php.net/manual/en/book.mbstring.php>`_ and `cURL
+<http://php.net/manual/en/book.curl.php>`_. To check which modules you
+have currently installed, run::
+
+  php -m
+
+To install with Linux/OSX:
```

Comment:
Not sure apt-get is available on macOS -- surely it is not by default. So, on macOS, it would be more like this (for MacPorts):

sudo port install phpXY-mbstring
sudo port install phpXY-curl

where XY is the PHP version currently installed on your system  (e.g., php72-curl).

I cannot find the package names for Homebrew -- maybe Homebrew PHP comes with mbstring and curl already installed.

Reply:
I will look into this further. While there were instructions for adding mbstring and curl in the README, I don't remember having to install them manually myself. I'll make sure what is there in the README is actually correct and necessary.

Reply:
It seems mbstring is included in Homebrew by default, but curl is not. 

Can you tell me more about openssl? There was no mention of it in the documentation previously, and we don't explicitly call any openssl functions. I want to be sure it is included properly. Would it be necessary on any installation no matter the OS, or just for mac?